File: src/test/java/com/Angus/StudentTest.java
package com.Angus;

import com.Angus.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

     @Test
    public void highScoreTest(){
        Student student = new Student("Angus", 80, 60);
        Assertions.assertEquals(80, student.highest());
    }

    @Test
    void getAverageTest(){
         Student student = new Student("Angus", 80, 60);
         Assertions.assertEquals(70, student.getAverage());
    }
}

File: src/test/kotlin/com/Angus/StudentTest.kt
package com.Angus

import com.Angus.Kotlin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest(){


    @Test
  fun gradingTest(){
        val student = Student("Angus", 80, 60)
        Assertions.assertEquals('C',student.grading())
    }


    @Test
    fun highestTest(){
        val student = Student("Angus2", 70, 60)
        Assertions.assertEquals(65, student.getAverage())
    }
}